mono/tests/Makefile.am

```diff
@@ -1031,6 +1031,7 @@ INTERP_DISABLED_TESTS = \
 	weak-fields.exe \
 	vararg.exe \
```

Comment:
I think `vararg.exe` should be removed then, right?

Reply:
@lewurm there are two varargs : vararg.cs and vararg.il. The point of the commit was to test them both.

Reply:
and both failing ofc